openstack/neutron/templates/deployment-rpc-server.yaml

```diff
@@ -40,7 +39,20 @@ spec:
           command: ['dumb-init', 'kubernetes-entrypoint']
           env:
             - name: COMMAND
-              value: "neutron-rpc-server --config-file /etc/neutron/neutron.conf --config-file /etc/neutron/plugins/ml2/ml2-conf.ini --config-file /etc/neutron/plugins/ml2/ml2-conf-aci.ini --config-file /etc/neutron/plugins/ml2/ml2-conf-manila.ini --config-file /etc/neutron/plugins/ml2/ml2-conf-arista.ini --config-file /etc/neutron/plugins/ml2/ml2-conf-asr1k.ini {{- if .Values.bgp_vpn.enabled }} --config-file /etc/neutron/networking-bgpvpn.conf{{- end }}{{- if .Values.fwaas.enabled }} --config-file /etc/neutron/neutron-fwaas.ini{{- end }}"
+              value: >-
+                neutron-rpc-server
+                --config-file /etc/neutron/neutron.conf
+                --config-file /etc/neutron/plugins/ml2/ml2-conf.ini
+                --config-file /etc/neutron/plugins/ml2/ml2-conf-aci.ini
+                --config-file /etc/neutron/plugins/ml2/ml2-conf-manila.ini
+                --config-file /etc/neutron/plugins/ml2/ml2-conf-arista.ini
+                --config-file /etc/neutron/plugins/ml2/ml2-conf-asr1k.ini
+                {{- if .Values.bgp_vpn.enabled }}
+                --config-file /etc/neutron/networking-bgpvpn.conf
+                {{- end }}
+                {{- if .Values.fwaas.enabled }}
+                --config-file /etc/neutron/neutron-fwaas.ini
+                {{- end -}}
```

Comment:
the -}} removes newlines at the end and makes the declaration invalid, see name: NAMESPACE: 
```
+               value: >-
+                 neutron-rpc-server
+                 --config-file /etc/neutron/neutron.conf
+                 --config-file /etc/neutron/plugins/ml2/ml2-conf.ini
+                 --config-file /etc/neutron/plugins/ml2/ml2-conf-aci.ini
+                 --config-file /etc/neutron/plugins/ml2/ml2-conf-manila.ini
+                 --config-file /etc/neutron/plugins/ml2/ml2-conf-arista.ini
+                 --config-file /etc/neutron/plugins/ml2/ml2-conf-asr1k.ini
+                 --config-file /etc/neutron/networking-bgpvpn.conf- name: NAMESPACE
```